src/NServiceBus.Metrics.ServiceControl/RingBufferExtensions.cs

```diff
@@ -4,15 +4,20 @@
 
 class RingBufferExtensions
 {
-    const int MaxExpectedWriteAttempts = 10;
+    // SpinWait.SpinOnce will start to yield at about 10 spinning iterations, so give it a few more tries before logging errors
+    const int MaxExpectedWriteAttempts = 15; 
+
     public static void WriteTaggedValue(RingBuffer buffer, string metricType, long value, int tag)
     {
-        var written = false;
+        var spinWait = new SpinWait();
         var attempts = 0;
-
-        while (!written)
+        
```

Comment:
nit: 🔥

Reply:
that new line was intentional, I think it makes the code more readable to space out local variable declarations from the while loop.

Reply:
I was not questioning the newline. I was questioning the spaces in front of the newline :)